renovate.json

```diff
@@ -14,7 +14,8 @@
     "**/charts/argo-events/Chart.yaml",
     "**/charts/argo-rollouts/Chart.yaml",
     "**/charts/argocd-image-updater/Chart.yaml",
-    "**/.github/workflows/renovate.yaml"
+    "**/.github/workflows/renovate.yaml",
+    "**/.github/workflows"
```

Comment:
Does that mean that we now enable renovate on all workflow files?

Reply:
Oh no. That's a bad commit. I'll remove when I'm at a computer.

Reply:
The badness has been removed now. Sorry about that.